pipeline-notebooks/pipeline-general.ipynb

```diff
@@ -726,6 +726,7 @@
     "    m_ocr_languages=[],\n",
     "    m_encoding=[],\n",
     "    m_xml_keep_tags=[],\n",
+    "    m_pdf_infer_table_structure = [],\n",
```

Comment:
Why is this an array?

Reply:
this is how we pass in self defined parameters in api tools repo. It can support multiple input like this example:
```
curl -X 'POST' \
  'https://api.unstructured.io/general/v0/general' \
  -H 'accept: application/json' \
  -H 'Content-Type: multipart/form-data' \
  -F 'files=@sample-docs/english-and-korean.png' \
  -F 'strategy=ocr_only' \
  -F 'ocr_languages=eng'  \
  -F 'ocr_languages=kor'  \
  | jq -C . | less -R
  ```
  so `m_ocr_languages=["eng", "kor"]` is  passing into the function this way (list of str)

Reply:
What potential values would be provided for the `pdf_infer_table_structure` parameter? I thought it was just a boolean.

Reply:
yep it is one boolean value, the value is obtained here: https://github.com/Unstructured-IO/unstructured-api/pull/139/files#diff-d80162975e0983c4067dc93f0d9a06f8867a7a982cf67a38975109e448d20c5eR217

Reply:
yeah, more context is unstructured-api-tools only allow for string array params right now, so that's why everything has the default array value

Reply:
Ah that's what I was missing and explains this. I didn't realize boolean values weren't getting converted at the moment. Thanks for that info!